proxy/InkAPI.cc

```diff
@@ -8941,3 +8942,110 @@ TSVConnReenable(TSVConn vconn)
     }
   }
 }
+
+// APIs for managing and using UUIDs.
+TSUuid
+TSUuidCreate(void)
+{
+  ATSUuid *uuid = new ATSUuid();
+  return (TSUuid)uuid;
+}
+
+TSReturnCode
+TSUuidDestroy(TSUuid uuid)
+{
+  sdk_assert(sdk_sanity_check_null_ptr((void *)uuid) == TS_SUCCESS);
+  ATSUuid *u = (ATSUuid *)uuid;
+
+  if (u->alive()) {
```

Comment:
If ``alive()`` is false you have a stale pointer right? At which point how do you know that it was really not alive?